regulations/static/regulations/js/source/views/drawer/toc-view.js

```diff
@@ -41,7 +41,7 @@ var TOCView = Backbone.View.extend({
     setActive: function(id) {
         var newActiveLink, subpart;
 
-        newActiveLink = this.$el.find('a[data-section-id="' + Helpers.findBaseSection(id) + '"]');
+        newActiveLink = this.$el.find('a[data-section-id="' + id + '"]');
```

Comment:
The caller is responsible for passing a `tocId` here. I'll update the signature. There's no conversion--we're passing the `tocId` through the dom in this proposal.